lib/iris/fileformats/netcdf.py

```diff
@@ -265,7 +265,10 @@ def __init__(self, shape, dtype, path, variable_name, fill_value):
     def ndim(self):
         return len(self.shape)
 
-    def __getitem__(self, keys):
+    def __getitem__(self, inkeys):
+        # Alter length one tuples to slice objects for safer indexing
+        keys = tuple([(slice(key[0], key[0] + 1) if isinstance(key, tuple) and
```

Comment:
I'd be more inclined to defer this to biggus (see my previous comment). For instance, I don't know if it needs to, but this wouldn't handle Ellipsis, and I suspect numpy array indexing etc. etc. 
Biggus has handled all of these cases already, so you don't have to...